Example/Example/ViewController.swift

```diff
@@ -15,7 +15,7 @@ final class ViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         
-        imageView.ok.setImage(with: URL(string: "https://www.gstatic.com/webp/gallery/4.webp")!)
+        imageView.ok.setImageUrl(with: URL(string: "https://www.gstatic.com/webp/gallery/4.webp")!)
```

Comment:
this doesn't read right to me. Either `imageView.ok.setImageUrl(URL(string:""))` or `imageView.ok.setImage(url: URL(string:""))`

Reply:
Ok, update to url. I don't like omitting the var name as that is less standard in swift syntax